projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -1264,6 +1281,32 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
     public set inlineEditorTemplate(template: TemplateRef<any>) {
         this._inlineEditorTemplate = template;
     }
+
+    /**
+     * Returns a reference to the validation error template.
+     * ```typescript
+     * let errorTemplate = this.column.errorTemplate;
+     * ```
+     */
+    @notifyChanges()
+    @WatchColumnChanges()
+    @Input('errorTemplate')
+    public get errorTemplate(): TemplateRef<any> {
+        return this._errorTemplate;
+    }
+    /**
+     * Sets the validation error template.
+     * ```html
+     * <ng-template igxCellValidationError let-cell="cell">
+     *     <div *ngIf="cell.errors?.['forbiddenName']">
+     *      This name is forbidden.
+     *     </div>
+     * </ng-template>
+     */
+    public set errorTemplate(template: TemplateRef<any>) {
+        this._errorTemplate = template;
```

Comment:
@MayaKirova Isn`t this an input setter ? Do we need to expose such thing? I mean this gives the possibility the error template to be defined outside the igx-column template. Is this why this is exposed? Also, if what I think is true maybe we should change the `html` doc

Reply:
All other template are exposed so we might as well expose this one as well. I'll update the html doc.